src/FormConfig/EmailConfig.php

```diff
@@ -22,8 +22,8 @@ public function __construct(RenderedSubmission $submission)
         $this->endpoint = $submission->getEndpoint();
         $message = \json_decode($submission->getMessage(), true);
 
-        $this->from = $message['from'];
-        $this->subject = $message['subject'];
+        $this->from = $message['from'] ?? 'noreply@elasticms.eu';
```

Comment:
do we want our hosting to be the default email address for everybody using the project?

Reply:
Fix #691e427